client/src/components/MyLibrary/index.js

```diff
@@ -1,264 +1,293 @@
+                        step='1'
+                        min='1'
+                        className='validate'
+                      ></input>
+                      <label id='library-label'>Minimum Players</label>
+                    </div>
+                    <div id='library-input' className='input-field col s4'>
+                      <input
+                        value={this.state.minPlaytime}
+                        name='minPlaytime'
+                        onChange={this.handleInputChange}
+                        type='number'
+                        step='1'
+                        min='1'
+                        className='validate'
+                      ></input>
+                      <label id='library-label'>{`Min Playtime (Minutes)`}</label>
+                    </div>
+                    <div id='library-input' className='input-field col s4'>
+                      <input
+                        value={this.state.maxPlaytime}
+                        name='maxPlaytime'
+                        onChange={this.handleInputChange}
+                        type='number'
+                        step='1'
+                        min='1'
+                        className='validate'
+                      ></input>
+                      <label id='library-label'>{`Max Playtime (Minutes)`}</label>
                     </div>
+                  </div>
+                  <div className='row'>
+                    <div id='library-input' className='input-field col s4'>
+                      <input
+                        value={this.state.minAge}
+                        name='minAge'
+                        onChange={this.handleInputChange}
+                        type='number'
+                        step='1'
+                        min='1'
+                        className='validate'
+                      ></input>
+                      <label id='library-label'>Minimum Age</label>
+                    </div>
+                    <div id='library-input' className='input-field col s4'>
+                      <select
+                        id='complexity'
+                        name='complexity'
+                        onChange={this.handleInputChange}
+                        value={this.state.complexity}
+                        className='validate'
+                      >
+                        <option
+                          className='complexity-placeholder'
+                          defaultValue=''
+                        >
+                          Complexity
+                        </option>
+                        <option value='Light'>Light</option>
+                        <option value='Medium'>Medium</option>
+                        <option value='Heavy'>Heavy</option>
+                      </select>
+                      <label id='library-label' htmlFor='complexity'></label>
+                    </div>
+                    <div id='library-input' className='input-field col s4'>
```

Comment:
This id is used multiple times, can't have that. `library-input` and so is `library-label`

Reply:
they're duplicated because they all need to apply to the label element and creating a className doesn't override the default materialize.css styling.